.github/workflows/build-devcontainer.yaml

```diff
@@ -38,7 +28,11 @@ jobs:
       - uses: docker/metadata-action@v4
         id: meta
         with:
-          images: ghcr.io/${{ github.repository_owner }}/prql-devcontainer-base
+          # We need to put `prql` in explictly because it needs to be lowercase;
+          # https://github.com/orgs/community/discussions/25768
+
+          # images: ghcr.io/${{ github.repository_owner }}/prql-devcontainer-base
+          images: ghcr.io/prql/prql-devcontainer-base
```

Comment:
`docker/metadata-action` should not require this processing because it normalizes image name.